topics/go/algorithms/searches/linearsearch/linearsearch.go

```diff
@@ -52,27 +53,33 @@ func doubleLinearSearchIterative(list []int, target int) int {
 
 	// Continue loop until leftIdx is smaller or equal to rightIdx value.
 	for leftIdx <= rightIdx {
-		if list[leftIdx] == target {
+
+		switch {
+
+		case list[leftIdx] == target:
 			return leftIdx
-		}
 
-		if list[rightIdx] == target {
+		case list[rightIdx] == target:
 			return rightIdx
+
+		default:
+			leftIdx++
+			rightIdx--
 		}
 
-		leftIdx++
-		rightIdx--
 	}
 
 	return -1
 }
 
-// doubleLinearSearchRecursive takes a sorted/random list of numbers and uses
-// the `recursive` process to check index by index to find the target in both
-// left and right index.
+// doubleLinearSearchRecursive takes a sorted/random list of numbers
+// and uses the `recursive` process to check index by index to find
+// the target in both left and right index.
 func doubleLinearSearchRecursive(list []int, target int, leftIdx int, rightIdx int) int {
 	if len(list) > 0 {
+
```

Comment:
Remove this line space and I will merge.

Reply:
I removed this line space and the line space inside the  `for` statement in line 56